Client/ApiClient.php

```diff
@@ -127,24 +128,33 @@ protected function getResponseData(
         Request $request
     ): ?string {
         $this->logger->debug(
-            "API Request:{$request->getMethod()} {$request->getUri()}",
+            "API Request",
             [
-                'body' => $request->getBody(),
```

Comment:
Ok maybe having the body is not a good idea

Reply:
The problem was that the getBody returns a stream, not a string, so if you watch bellow, the body is still here ;)

Reply:
Yes I noticed, I'm ok with removing it